Sasquatch/Sasquatch/ViewControllers/MSDocumentDetailsViewController.swift

```diff
@@ -72,6 +74,32 @@ class MSDocumentDetailsViewController: UIViewController, UITableViewDelegate, UI
     self.presentingViewController?.dismiss(animated: true, completion: nil)
   }
 
+  @IBAction func refreshButtonClicked(_ sender: Any) {
+    var partition = documentContent?.partition ?? ""
+    if (partition.contains(appCenter.kMSDataUserDocumentsPartition)) {
+        partition = appCenter.kMSDataUserDocumentsPartition
+    }
+    self.appCenter.readDocumentWithPartition(partition, documentId: documentContent?.documentId ?? "", documentType: MSDictionaryDocument.self, completionHandler: { (document) in
+        self.documentContent = document
+        DispatchQueue.main.async {
+            self.tableView.reloadData()
+        }
+        
+        if (partition == self.appCenter.kMSDataUserDocumentsPartition) {
```

Comment:
```suggestion
        if ([self.appCenter.kMSDataUserDocumentsPartition isEqualToString partition]) {
```

Reply:
My understanding is that `isEqualToString` is a function in Objective C and the correct way to check string equality in Swift is to use `==`

Reply:
Oh yeah my bad. Forgot this is swift